Resources/Prototypes/Datasets/tips.yml

```diff
@@ -32,30 +28,99 @@
   - "The AME is a high-priority target and is easily sabotaged. Make sure to set up the Singularity or Solars so that you don't run out of power if it blows."
   - "You can add labels to any item, including food or pill canisters, using a hand labeler."
   - "Riot armor is significantly more powerful against opponents that aren't using guns compared to regular armor."
-  - "As a Security Officer, remember that correlation does not equal causation. Someone may have just been at the wrong place at the wrong time!"
   - "As a ghost, you can use the Verb Menu to orbit around and follow any entity in game automatically."
+  - "As a Mouse, you can alt-click food to eat it."
```

Comment:
Anyone can alt-click food to eat it, not just mice.

Reply:
fixed